templates/cli/lib/commands/deploy.js.twig

```diff
@@ -41,6 +41,7 @@ const {
 
 const STEP_SIZE = 100;      // Resources
 const POOL_DEBOUNCE = 2000; // Milliseconds
+const POOL_MAX_DEBOUNCE = 30; // Times
```

Comment:
So I think this should be renamed to `POLL_` and not `POOL` lets also refactor the existing variable